src/app/screens/sendBtc/recipientSelector.tsx

```diff
@@ -1,39 +1,26 @@
+import InputScreen from '@components/inputScreen';
 import useWalletSelector from '@hooks/useWalletSelector';
 import { validateBtcAddress } from '@secretkeylabs/xverse-core';
 import Button from '@ui-library/button';
 import Input from '@ui-library/input';
-import { useMemo, useState } from 'react';
+import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import styled from 'styled-components';
 
-const Container = styled.div`
-  flex: 1 1 100%;
-
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-`;
-
-const Buttons = styled.div`
-  margin: ${(props) => props.theme.spacing(12)}px 0;
-`;
-
-type Props = {
+type BtcRecipientScreenProps = {
   recipientAddress: string;
   setRecipientAddress: (address: string) => void;
   onNext: () => void;
   isLoading: boolean;
   header?: React.ReactNode;
 };
 
-// TODO: this could be extracted into a component for reuse
```

Comment:
I think this should still be extracted into a generic recipient selector component. Maybe I can do it in the base branch and you can make use of it for stacks in this PR?

Reply:
Sure, sounds good. [It'll solve this comment as well.](https://github.com/secretkeylabs/xverse-web-extension/pull/767#discussion_r1480977803)